src/components/AlphabetFilter/AlphabetFilter.stories.mdx

```diff
@@ -0,0 +1,141 @@
+      table: { defaultValue: { summary: false } },
+    },
+    onClick: { control: false },
+  }}
+/>
+
+# AlphabetFilter
+
+| Component Version | DS Version |
+| ----------------- | ---------- |
+| Added             | `1.2.0`    |
+| Latest            | `1.2.0`    |
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Component Props](#component-props)
+- [Accessibility](#accessibility)
+- [Examples](#examples)
+
+## Overview
+
+<Description of={AlphabetFilter} />
+
+The `AlphabetFilter` component provides a method for alphabetical filtering on a list of items.
+
+### Consuming App Responsibilities
+
+When the `onClick` handler is executed, it will be up to the consuming app to execute filtering
+updates in real-time on the client-side or to perform a full page refresh.
+
+When the `Show All` button is clicked, all filtering should be removed from the list of items that
+the component is addressing and the full list of items should be displayed.
+
+## Component Props
+
+<Canvas withToolbar>
+  <Story
+    name="AlphabetFilter with Controls"
+    args={{
+      activeLetters: undefined,
+      className: undefined,
+      currentLetter: undefined,
+      descriptionText: "This is description text.",
+      headingText: "AlphabetFilter",
+      id: "alphabet-filter-id",
+      isDisabled: false,
+      onClick: undefined,
+    }}
+  >
+    {(args) => <AlphabetFilter {...args} />}
+  </Story>
+</Canvas>
+
+<ArgsTable story="AlphabetFilter with Controls" />
+
+## Accessibility
+
+The `AlphabetFilter` component renders as an HTML `<nav>` element with a WAI-ARIA role
+of `role="navigation"`. Each letter button has `aria-label="Page {#}"` applied, where
+{#} is equal to a button's letter value. The Show All button has `aria-label="Show All"`
+applied. When a letter button is set to a current state, `aria-current="page"` is applied
+to the button and removed from all other letter buttons.
+
+The `AlphabetFilter` component is accessible via keyboard. The color contrast between
+foreground color and background color is 4.5:1. If text size is 200%, the
+button scales with text so there is no overlap.
+
+### Consuming App Accessibility
+
+If filter results are refreshed on the client-side without a full page refresh, the
+updated filter results must be the child of an element that has the `role=”alert”` attribute
+applied or as a child of a Live Region container with the `aria-live="polite"` attribute applied.
+
+## Examples
+
+### AlphabetFilter with activeLetters set
+
+Pass an array of letters to narrow filter options.
+
+Ex. `["#", "a", "c", "d", "e", "h", "l", "m", "n", "p", "r", "s"]`
+
+<Canvas>
+  <DSProvider>
+    <AlphabetFilter
+      activeLetters={[
+        "#",
+        "a",
+        "c",
+        "d",
+        "e",
+        "h",
+        "l",
+        "m",
+        "n",
+        "p",
+        "r",
+        "s",
+      ]}
+      onClick={undefined}
+    />
+  </DSProvider>
+</Canvas>
+
+### Set letter selection programmatically
+
```

Comment:
Add more context such as, "Use the `currentLetter` prop to pass a single letter value to set the selected letter page when the component first renders, or to programmatically set the current letter."

This example shows how the initial selected letter works but maybe we need another example where the `currentLetter` is updated dynamically by the user. Check out the `Select` examples at the end https://nypl.github.io/nypl-design-system/reservoir/v1/?path=/docs/components-form-elements-select--select-with-controls where there are controlled and uncontrolled examples. This will also help show how the `onClick` function should be used by the consuming app.

Let me know if this makes sense. It is more involved on our side, but it'll give a better example to users on one particular implementation.